pyproject.toml

```diff
@@ -101,7 +101,7 @@ types-pycurl = "*"
 types-PyYAML = "*"
 types-requests = "*"
 types-tabulate = "*"
-pyqt5 = "*"
+pyqt5 = { version = "*", markers = "platform_machine == 'x86_64'" }
```

Comment:
when we write exceptions like this, make sure to leave a good comment explaining why so it's easy to resolve later